### PR TITLE
Make build-registry work correctly when ember-data addon is used

### PR DESCRIPTION
**Pull request by pangratz**
Credit for fix code goes to @rwjblue, I am merely the humble PR creator.

This addresses emberjs/data#4071.

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -98,6 +98,18 @@ module.exports = {
...
+      name: 'ember-data-2.3',
+      dependencies: {
+        "ember": "components/ember#release",
+      },
+      devDependencies: {
+        "ember-data": "~2.3"
+      },
+      resolutions: {
+        "ember": "release"
+      }
```
Comment (pangratz) [inline config/ember-try.js]: Hmm, I just realized that ember-try only works with bower dependencies for now. So I am not sure this config is really helpful to test the code path below :pensive:

Comment (rwjblue) [inline config/ember-try.js]: Agreed, but it doesn't hurt...

**Comment by rwjblue** [comment]
Thank you! I'll release shortly...

**Comment by seawatts** [comment]
@rwjblue just curious, do you have an ETA? Also will this have to kick off a release in ember-quint and ember-cli-qunit as well?

**Pull request merged**